osgp/protocol-adapter-iec61850/osgp-protocol-adapter-iec61850/src/main/java/org/opensmartgridplatform/adapter/protocol/iec61850/infra/messaging/DeviceResponseMessageSender.java

```diff
@@ -112,7 +114,7 @@ private void sendMessage(final ProtocolResponseMessage responseMessage) {
                 responseMessage.getCorrelationUid(), responseMessage.getDeviceIdentification(),
                 responseMessage.getMessageType(), responseMessage.getMessagePriority());
 
-        this.iec61850ResponsesJmsTemplate.send(new MessageCreator() {
+        this.jmsTemplate.send(new MessageCreator() {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Reduce this anonymous class number of lines from 23 to at most 20, or make it a named class. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://ci.opensmartgridplatform.org/sonarqube/coding_rules#rule_key=squid%3AS1188)